### ISSUE TITLE
Default loss choice for ADDA

### ISSUE THREAD

**Issue by saadabbasi** [adapt-python/adapt#10]
I've been reading the source for `ADDA` and the base `BaseDeepFeature` class. I note that the default loss function is `MSE`. Is this loss function used for training the source model or the discriminator? As far as I understand, the discriminator is a classifier and is typically trained with binary cross-entropy loss.

I am hoping to use ADDA for a regression problem but was wondering how to modify the technique. A few papers briefly mention using MSE for ADDA but I am not sure where this loss function is used - for source or discriminator?
